diff -Naur inkscape-0.48.3.1.orig/share/extensions/text_braille.py inkscape-0.48.3.1/share/extensions/text_braille.py
--- inkscape-0.48.3.1.orig/share/extensions/text_braille.py	2013-01-15 02:11:31.461801844 +0700
+++ inkscape-0.48.3.1/share/extensions/text_braille.py	2013-01-15 02:13:33.801809524 +0700
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-
+# -*- coding: utf8 -*-
 #encoding: utf-8
 import chardataeffect, inkex, string
 
